openwisp_monitoring/monitoring/migrations/0010_metric_configuration.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.3 on 2020-05-21 22:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0009_allow_float_threshold'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='metric',
+            name='configuration',
+            field=models.CharField(
+                choices=[
```

Comment:
do it like it's done here please:

https://github.com/openwisp/openwisp-monitoring/blob/c4defa4773f1fc11d5c99ff94dcf0983c9af3ba4/openwisp_monitoring/monitoring/migrations/0006_add_configuration.py#L7-L21